collie/model/base/base_pipeline.py

```diff
@@ -767,7 +729,8 @@ def get_user_predictions(self,
         if item_id > self.hparams.num_items:
             raise ValueError(
                 f'``item_id`` {item_id} is not in the model. '
-                f'Expected id between 0 and {self.hparams.num_items}'
+                'Expected ID between ``0`` and ``self.hparams.num_items``, '
+                f'{self.hparams.num_items}'
```

Comment:
Same optional comment applies here too if you decide to implement it. If not, no worries.